practice_app/pages/index.js

```diff
@@ -86,35 +84,94 @@ export default function Home({ context }) {
     const onConvertButtonClick = async () => {
         try {
             const ipdata = await getUsersIP()
-            const pricedata = await priceConverter(10, ipdata.query, "USD")
+            const pricedata = await priceConverter(10, ipdata.query, 'USD')
             Promise
-            setPriceConversionText(`Your ip adress is ${ipdata.query} and currency of the country that this IP belongs to is ${pricedata.currency}.
-            If price of a product is 10 USD, then it is equal to ${pricedata.price.toFixed(2)} ${pricedata.currency} in your currency.`)
+            setPriceConversionText(`Your ip adress is ${
+                ipdata.query
+            } and currency of the country that this IP belongs to is ${pricedata.currency}.
+            If price of a product is 10 USD, then it is equal to ${pricedata.price.toFixed(2)} ${
+                pricedata.currency
+            } in your currency.`)
         } catch (error) {
             console.log(error)
-            setPriceConversionText("Sorry, service unavailable at this time.")
+            setPriceConversionText('Sorry, service unavailable at this time.')
         }
     }
 
     // Returns the IP of the user
     const getUsersIP = async () => {
-        const {data} = await axios.get("http://ip-api.com/json/")
+        const { data } = await axios.get('http://ip-api.com/json/')
         return data
     }
 
     // Using the given ip, finds the currency of the country that the given IP belongs to.
-    // Then, converts the given price in srcCurrency to the new currency. 
+    // Then, converts the given price in srcCurrency to the new currency.
     // Returns the result
     const priceConverter = async (pr, ip, srcCurrency) => {
         try {
-            const {data} = await axios.get('api/getConvertedPrice?ip='+ip+'&price='+pr+'&srcCurrency='+srcCurrency)
+            const { data } = await axios.get(`api/getConvertedPrice?ip=${ip}&price=${pr}&srcCurrency=${srcCurrency}`)
             return data
-            
         } catch (error) {
-            console.error(error) 
-        } 
+            console.error(error)
+        }
+    }
+    // This function returns the top 10 countries with maximum number of deaths and their death statistics
+    const getCountriesWithMaxDeaths = async () => {
+        // Get the data from our api
+        const { data } = await axios.get('api/getCountriesWithMaxDeaths')
+        // Print to the front-end
+        const ctx = document.getElementById('bar_deaths')
+        ctx.style.visibility = 'visible'
+        const chart = new Chart(ctx, {
+            // We are creating a bar chart for death statistics of top 10 countries
+            type: 'bar',
+            data: {
+                labels: data.keys,
+                datasets: [
```

Comment:
Although this part might work, mixing pure javascript HTML manipulation with React/Next.js will give hard to solve errors and potentially unpredictable behavior. If you have the time, change this to use `react-chartjs-2` instead. It won't be a big change since you should be able to pass the entire chart configuration directly.